### PR TITLE
Feature creation

### PR DESCRIPTION
**Pull request by OleksiiKolomiiets**

### CODE REVIEW

File: DataStructuresMemories/Models/DataStructuresModel.swift [hunk 1]
```diff
@@ -60,15 +60,15 @@ class DataStructModel: DataStruct {
     func getDescription(of structure: String) -> String {
         guard let currentDataStruct = DataStructuresNames(rawValue: getCurrentIndex(of: structure))
         else {
-            fatalError("Name: \(structure) does not match any name of: DataStructuresNames")
+            fatalError("Name: \(structure) does not match any name of: DataStructuresNames.")
```
Comment (AlexeyTyurenkov) [inline DataStructuresMemories/Models/DataStructuresModel.swift]: Wow. would you kill baby who doesn't say you hello? 
fatalError is from another tale, please avoid using it in real projects

File: DataStructuresMemories/ViewControllers/DataStructViewController.swift [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+//  Copyright © 2018 Oleksii Kolomiiets. All rights reserved.
+//
+
+import UIKit
+
+class DataStructViewController: UIViewController {
+    
+    @IBOutlet weak var textInsideLable: UILabel!
+    
+    var tappedCell: DataStructModel?
```
Comment (AlexeyTyurenkov) [inline DataStructuresMemories/ViewControllers/DataStructViewController.swift]: why don't we use protocol there?

File: DataStructuresMemories/ViewControllers/DataStructuresTableViewController.swift [hunk 3]
```diff
@@ -10,9 +10,9 @@ import UIKit
 
 
 
-class DataStructuresMemoriesTableViewController: UITableViewController {
+class DataStructuresTableViewController: UITableViewController {
    
-    let dataStructModel = DataStructModel()
+    let dataStructModel = DataStructuresModel()
```
Comment (AlexeyTyurenkov) [inline DataStructuresMemories/ViewControllers/DataStructuresTableViewController.swift]: It is data source, so could you name it properly?

**Review by AlexeyTyurenkov — CHANGES_REQUESTED** [review]

File: DataStructuresMemories/Models/DataStructuresModel.swift (hunk 1 above)
Comment (OleksiiKolomiiets) [reply DataStructuresMemories/Models/DataStructuresModel.swift]: Got it)

File: DataStructuresMemories/ViewControllers/DataStructuresTableViewController.swift (hunk 3 above)
Comment (OleksiiKolomiiets) [reply DataStructuresMemories/ViewControllers/DataStructuresTableViewController.swift]: Yes. I will change it right now.

File: DataStructuresMemories/ViewControllers/DataStructViewController.swift (hunk 2 above)
Comment (OleksiiKolomiiets) [reply DataStructuresMemories/ViewControllers/DataStructViewController.swift]: I agree with you, it will be much better to use protocol there.

**Pull request merged**